### ISSUE TITLE
Pressure Plate issue newest CMI version

### ISSUE THREAD

**Issue by muddup** [Zrips/CMI#2669]
**Description of issue or feature request:**

So updated from 8.6.18.1 to 8.6.18.2 and when I step on a pressure plate attached to a door i get this. https://pastebin.com/ADDHKDGS I replaced CMI with previous version and this error does not happen.

#### DELETE THIS SECTION IF YOU ARE SUBMITTING A FEATURE REQUEST ####

**ERROR (DELETE IF YOU HAVE NO ERROR):**
```
####################
## PASTE ERROR HERE ##
####################
https://pastebin.com/ADDHKDGS

**CONFIG SECTION (DELETE IF NOT RELEVANT):**
```
#####################################
#####################################
```

---

**Cmi Version :** CMI 8.6.18.2 

**Server Type (Spigot/Paperspigot/etc):** Paper 1.15.1 # 58

**Server Version (using `/ver`):** 
 Paper version git-Paper-58 (MC: 1.15.1) (Implementing API version 1.15.1-R0.1-SNAPSHOT)

**Comment by mrfloris** [comment]
I believe I had reported the same issue and the .3 release might fix this for you as well. 
Try the upgrade and let us know if it still happens.

**Comment by Zrips** [comment]
This got fixed with 8.6.18.3 hotfix release
